deployments/docker-compose/metrics/envoy.yaml

```diff
@@ -0,0 +1,356 @@
+                                {
+                                  "name": "hits",
+                                  "delta": 1
+                                }
+                              ]
+                            },
+                            {
+                              "method": "GET",
+                              "pattern": "/o{*}c",
+                              "usages": [
+                                {
+                                  "name": "oidc",
+                                  "delta": 1
+                                },
+                                {
+                                  "name": "Hits",
+                                  "delta": 1
+                                }
+                              ]
+                            },
+                            {
+                              "method": "any",
+                              "pattern": "/{anything}?bigsale={*}",
+                              "usages": [
+                                {
+                                  "name": "sale",
+                                  "delta": 5
+                                }
+                              ]
+                            }
+                          ]
+                        }
+                      ],
+                      "passthrough_metadata": true
+                    }
+                fail_open: false
+          - name: envoy.filters.http.wasm
+            typed_config:
+              "@type": type.googleapis.com/udpa.type.v1.TypedStruct
+              type_url: type.googleapis.com/envoy.extensions.filters.http.wasm.v3.Wasm
+              value:
+                config:
+                  name: "cache_filter"
+                  root_id: "cache_filter"
+                  configuration: 
+                    "@type": type.googleapis.com/google.protobuf.StringValue
+                    value: |
+                      {}
+                  vm_config:
+                    runtime: "envoy.wasm.runtime.v8"
+                    vm_id: "my_vm_id"
+                    code:
+                      local:
+                        filename: "/usr/local/bin/cache_filter.wasm"
+                    configuration: {}
+                    allow_precompiled: true
+          - name: envoy.filters.http.router
+            typed_config: {}
+  clusters:
+  - name: backend_service
+    connect_timeout: 0.25s
+    type: strict_dns
+    lb_policy: round_robin
+    load_assignment:
+      cluster_name: backend_service
+      endpoints:
+      - lb_endpoints:
+        - endpoint:
+            address:
+              socket_address:
+                address: backend_service
+                port_value: 8000
+  
+  - name: outbound|443||su1.3scale.net
+    connect_timeout: 5s
+    type: logical_dns
+    lb_policy: round_robin
+    upstream_connection_options:
+      tcp_keepalive: {}
+    load_assignment:
+      cluster_name: outbound|443||su1.3scale.net
```

Comment:
Just to avoid confusion you might want to replace `su1.3scale.net` with `listener` across the whole configuration.

Reply:
will fix.